src/table/components/__tests__/TableTotals.spec.tsx

```diff
@@ -16,6 +16,7 @@ describe('<TableTotals />', () => {
   let layout: TableLayout;
   let selectionsAPI: ExtendedSelectionAPI;
   let keyboard: stardust.Keyboard;
+  let isSelectionMode: boolean;
```

Comment:
Seems like you are not changing this

Reply:
or just `isSelectionMode={false}` below?

Reply:
Yes either of them 😄